GOVERNANCE.md

```diff
@@ -0,0 +1,76 @@
+
+## Governance structure
+
+The below chart describes the governance structure.
+
+```mermaid
+graph TB
+style lfegb fill:white,color:black;
+style lfesssc fill:white,color:black;
+lfegb[LF Energy Governing Board]---lfesssc[LFESS Steering Committee]
+
+subgraph LFESS["LF Energy Standards and Specifications"]
+style LFESS fill:#1063a1,color:white,text-align:left;
+style cdscwg1 fill:white,color:black;
+style cdscwg2 fill:white,color:black;
+style samwg1 fill:white,color:black;
+style samwg2 fill:white,color:black;
+lfesssc---cdscwg1
+lfesssc---cdscwg2
+lfesssc---samwg1
+lfesssc---samwg2
+
+subgraph cdsc["Carbon Data Specifications Consortium (CDSC)"]
+style cdsc vertical-align:bottom,color:white,fill:#ff8300;
+cdscwg1[Power Systems WG]
+cdscwg2[Customer Data WG]
+end
+
+subgraph sam["Super Advanced Meter (SAM)"]
+style sam vertical-align:bottom,color:white,fill:#006b7a
+samwg1[Architecture WG]
+samwg2[Use Cases WG]
+end
+
+end
+```
+
+### LF Energy Governing Board
+
+The role of the LF Energy Governing Board is to provide fiduciary and legal oversight for LFESS. Specifically, it has these primary roles.
+
+- Define the process and requirements for Steering Membership ( detailed at https://github.com/lf-energy/foundation/blob/main/lfess-steering-committee-appointment-policy.md ).
+- Handle funding and infrastructure requests and incorporate those into the overall budget of LF Energy.
+
+### LFESS Steering Committee
+
+The LFESS Steering Committee provides oversight of the working groups within LFESS, with the primary role of ensuring the working groups have the resources needed to build the specifications. Specific duties of the LFESS Steering Committee include:
+
+- Approve the creation of new working groups
+- Maintain a lifecycle for working groups
+- Approve specification publication
+- Assess any resource or infrastructure needs for the working groups, escalating them to the [LF Energy Governing Board][#lf-energy-governing-board] as needed.
+
+## Participating and contributing
+
+The working groups are open to anyone, and there are no financial requirements, or corporate affiliations needs for participation. Participation includes signing up for mailing lists, joining Slack channels, or participating in a working group or community meeting. These groups are open and transparent; each group defines how to participate in their individual working group repositories.
+
+If someone wants to contribute to a specification, then a [membership](#membership) is required. Contributing to a specification is defined as anything said or written by a person that would be incorporated into a specification. Examples could include:
+
+- A new issue or pull request opened requesting a change to a specification
+- Comments on an issue or pull request that is making a change to a specification.
+- Comments shared during a meeting that are directly incorporated into a specification.
+
```

Comment:
"Information on how to participate in the working groups can be found here<link>"

Reply:
Would this be helpful? can we link to the respective projects?

Reply:
I like the idea of a seperate document that catalogs all the projects; let me add that.